.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Default repo owners
-*       @BearHanded @braxex
+-       @BearHanded @braxex
```

Comment:
```suggestion
*       @BearHanded @braxex
```

Reply:
I recall using - broke the code owners file awhile back, its been a bit though so I could be mistaken